src/pages/index.js

```diff
@@ -164,41 +164,6 @@ function Home() {
             </div>
           </section>
         )}
-        <div className={clsx("hero hero--secondary", styles.heroBanner)}>
```

Comment:
None of these three buttons have a 0.25 page! (Technically one of them does but it just says "TODO"!) 

So I removed this whole green block.